src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -250,23 +252,23 @@ public void run() {
     driver.run();
   }
 
-  private boolean launchNode(SchedulerDriver driver, Offer offer,
-        String nodeName, List<String> taskTypes, String executorName) {
+  private void launchNode(SchedulerDriver driver, Offer offer,
+        String nodeName, Map<String,String> taskInfo, String executorName) {
```

Comment:
TaskInfo is already a different, specific protobuf. Can we get a new name for this variable? Maybe "taskTypeToNameMap" to be super descriptive, or "tasksToLaunch"?